### PR TITLE
Extra test case when indexing an array of functions

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

File: pass/Easy.cpp [hunk 1]
```diff
@@ -128,6 +128,17 @@ namespace easy {
           }
         }
       }
+      if(auto *GEP = dyn_cast<GetElementPtrInst>(V)) {
+        return mayAlias(GEP->getPointerOperand(), GO);
+      }
+      if(auto* OtherGV = dyn_cast<GlobalVariable>(V)) {
+        if(!OtherGV->hasInitializer())
+          return false;
+        return mayAlias(OtherGV->getInitializer(), GO);
```
Comment (jmmartinez) [inline pass/Easy.cpp]: Similar to the Alloca case, shouldn't we also look the StoreInst that take OtherGV as the pointer parameter?

File: tests/simple/fun_ptr_c.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+int main(int argc, char** argv) {
+
+  static void (*(come_and_get_some[]))(void*dat) = {foo, bar};
+
+  easy::FunctionWrapper<void(void*, unsigned, unsigned)> map_w = easy::jit(map, _1, _2, _3, come_and_get_some[argc?1:0], easy::options::dump_ir(argv[1]));
+
+  int data[] = {1,2,3,4};
+  map_w(data, sizeof(data)/sizeof(data[0]), sizeof(data[0]));
```
Comment (jmmartinez) [inline tests/simple/fun_ptr_c.cpp]: This is great!

**Comment by serge-sans-paille** [comment]
Indeed

**Comment by serge-sans-paille** [comment]
@jmmartinez updated to handle stores to a GV

**Pull request merged**